Weather/WeatherObserved/harvest/spain_weather_observed_harvest.py

```diff
@@ -311,7 +320,20 @@ def setup_logger():
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Station Codes to be harvested')
+    parser.add_argument('stations', metavar='stations', type=str, nargs='*', help='Station Code')
```

Comment:
how can i know which codes are available?
how to separate the codes? comma, space?

Reply:
here you can find the concerned weather statiosn

https://jmcanterafonseca.carto.com/viz/e7ccc6c6-9e5b-11e5-a595-0ef7f98ade21/public_map

station codes are separated with spaces, this typical argparse Python confoguration